Dockerfile

```diff
@@ -285,19 +285,19 @@ FROM base as auto-gptq-cache
 COPY --from=auto-gptq-installer /usr/src/auto-gptq-wheel /usr/src/auto-gptq-wheel
 
 
-## Final Inference Server image ################################################
-FROM cuda-runtime as server-release
+## Full set of python installations for server release #########################
+
+FROM cuda-runtime as python-installations
+
 ARG PYTHON_VERSION
 ARG SITE_PACKAGES=/opt/tgis/lib/python${PYTHON_VERSION}/site-packages
 
-# Install C++ compiler (required at runtime when PT2_COMPILE is enabled)
-RUN dnf install -y gcc-c++ git && dnf clean all \
-    && useradd -u 2000 tgis -m -g 0
-
+# TODO: comment why this is required
```

Comment:
Anybody have context as to why the shell needed to be specified as `/bin/bash -c $cmd`?

(I could go run the build without it and see what fails, it just takes a while without the build cache from main)

Reply:
I expect that it is not needed anymore. It may have been here due to the use of `conda` if, at some point, we relied on modifications to `~/.bashrc` to activate the environment. But we set the `PATH` ourselves now and that seems to be sufficient.

Reply:
Yup, seems to work fine and pass GPU tests without it. Thanks @tjohnson31415!